static/_redirects

```diff
@@ -281,6 +281,7 @@
 /docs/tasks/run-application/podpreset/     /docs/tasks/inject-data-application/podpreset/ 301
 /docs/tasks/run-application/update-api-object-kubectl-patch/    /docs/tasks/manage-kubernetes-objects/update-api-object-kubectl-patch/ 301
 /docs/tasks/stateful-sets/deleting-pods/     /docs/tasks/run-application/force-delete-stateful-set-pod/ 301
+/ja/docs/tasks/tools/install-minikube/     https://minikube.sigs.k8s.io/docs/start/ 302
```

Comment:
Hi @mox692 .
I checked that `/docs/tasks/tools/install-minikube/` redirects to the `minikube` docs.
Does `/ja/docs/tasks/tools/install-minikube/` redirect?

Page preview:
https://deploy-preview-26495--kubernetes-io-master-staging.netlify.app/ja/docs/tutorials/hello-minikube/

Otherwise, the other links look correct.

Thanks!

Reply:
To check redirect is working, I tried to build website locally, but not succeed.

```
$ make container-image
docker build . \
		--network=host \
		--tag kubernetes-hugo:v0.74.3-1ead0bb5edad \
		--build-arg HUGO_VERSION=0.74.3
Sending build context to Docker daemon  3.584kB
Step 1/8 : FROM alpine:latest
 ---> e50c909a8df2
Step 2/8 : LABEL maintainer="Luc Perkins <lperkins@linuxfoundation.org>"
 ---> Using cache
 ---> 54c07b406044
Step 3/8 : RUN apk add --no-cache     curl     git     openssh-client     rsync     build-base     libc6-compat     npm &&     npm install -D autoprefixer postcss-cli
 ---> Using cache
 ---> 9f5c37054fec
Step 4/8 : ARG HUGO_VERSION
 ---> Using cache
 ---> d0ef7dda007e
Step 5/8 : RUN mkdir -p /usr/local/src &&     cd /usr/local/src &&     curl -L https://github.com/gohugoio/hugo/releases/download/v${HUGO_VERSION}/hugo_extended_${HUGO_VERSION}_Linux-64bit.tar.gz | tar -xz &&     mv hugo /usr/local/bin/hugo &&     addgroup -Sg 1000 hugo &&     adduser -Sg hugo -u 1000 -h /src hugo
 ---> Running in d6ea0acf9102
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
  0     0    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0curl: (6) Could not resolve host: github.com
tar: invalid magic
tar: short read
The command '/bin/sh -c mkdir -p /usr/local/src &&     cd /usr/local/src &&     curl -L https://github.com/gohugoio/hugo/releases/download/v${HUGO_VERSION}/hugo_extended_${HUGO_VERSION}_Linux-64bit.tar.gz | tar -xz &&     mv hugo /usr/local/bin/hugo &&     addgroup -Sg 1000 hugo &&     adduser -Sg hugo -u 1000 -h /src hugo' returned a non-zero code: 1
```

I haven't investigated this error yet. I'll check this later.
If hurry, Could anyone who can build website locally check the redirect instead?

Thank you:)

Reply:
The error says tar ball of hugo downloading fails so maybe you can retry?

Reply:
Hi @mox692.
Is the plan to merge these changes to `dev-1.18-ja.2` and then merge this branch (along with other changes) to `release-1.18` or possibly `master`? If so, I think it is okay to merge these changes in.

Reply:
Assuming the `ja` localization team approves, feel free to remove the hold.
/approve
/hold